packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.spec.md

```diff
@@ -0,0 +1,130 @@
+# Breadcrumb + Breadcrumb Item
+
+## Overview
+A `fast-breadcrumb` component is used as a navigational aid, allowing users to maintain awareness of their locations within a program, app, or a website.
+
+### Use Cases
+- James is browsing an ecommerce website and is looking for a few items to add to his new kitchen. He needs a knife set, and an espresso machine. He checks out `Home and Kitchen` page, then selects `Kitchen and Dining`, first category he picks is `Cutlery`, he finds the sets of knives. There is now a breadcrumb trail of the pages James has clicked through. `Home and Kitchen /  Kitchen and Dining / Cutlery / Knife Sets`
+He clicks on `Kitchen and Dining` to go back to the category of kitchen items, selects `Coffee and Espresso` finds the best one that suits his needs. If James wants to continue looking for another item for his kitchen, or other things for his home, he just needs to click on any of the links in the breadcrumb trail to continue his search.
+
+### Prior Art/Examples
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Atlaskit](https://atlaskit.atlassian.com/packages/core/breadcrumbs)
+- [Carbon Design](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [FAST Breadcrumb (React)](https://www.npmjs.com/package/@microsoft/fast-components-react-msft)
```

Comment:
I'm not sure this is the right path anymore, this link takes you to an npmjs page that says the package has been deprecated and the documentation link on that page doesn't work. I'm not sure what the right link would be though.

Reply:
I mean, it is deprecated and technically correct in terms of pointing to this. If you want a new link to the code: https://github.com/microsoft/fast-react/tree/master/packages/react/fast-components-react-msft/src/breadcrumb

Reply:
thanks. I will use the new link